frontend/src/components/pages/ManageStoryTranslationPage.tsx

```diff
@@ -133,23 +133,17 @@ const ManageStoryTranslationPage = () => {
   };
 
   const callUpdateStoryMutation = async () => {
-    if (
```

Comment:
nit: i dont think it hurts to keep this if statement in incase someone tries to use it in the future